helpers/meetupr_server_setup.R

```diff
@@ -0,0 +1,27 @@
+
+# One-Time Key Encryption Setup -------------------------------------------
+
+### Create a key
+
+token_path <- "~/Library/Application Support/meetupr/meetup_token.rds"
```

Comment:
Use file.path to build the path. 

Use path.expand to expand ~

Reply:
Done, thanks